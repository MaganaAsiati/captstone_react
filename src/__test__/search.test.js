import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import store from '../Redux/configureStore';
import SearchBar from '../components/search/search';

describe('SearchBar Component test', () => {
  it('Renders search Row', () => {
    const search = '';
    const handleOnChange = () => {};
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <SearchBar handleOnChange={handleOnChange} value={search} />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
