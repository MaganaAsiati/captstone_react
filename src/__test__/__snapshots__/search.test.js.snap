// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Component test Renders search Row 1`] = `
<div
  className="searbar"
>
  <form>
    <input
      className="input-search"
      onChange={[Function]}
      placeholder="search a country..."
      type="search"
      value=""
    />
  </form>
</div>
`;
